pomdp_py/algorithms/po_uct.pyx

```diff
@@ -295,6 +295,20 @@ cdef class POUCT(Planner):
         """
         self._rollout_policy = rollout_policy
 
+    cpdef QNode _create_qnode(
+        self,
```

Comment:
This lgtm. Please add docstring though; Looks like it's necessary to create QNode with parameters that differ from default ones e.g. `self._num_visits_init` in CC-POMCP. Why?